src/Console/Command/DebugCommand.php

```diff
@@ -63,4 +69,43 @@ private function showEvents(OutputStyle $console): void
             array_map(null, array_keys($events), array_values($events)),
         );
     }
+
+    private function showSubscribers(OutputStyle $console): void
+    {
+        $eventNames = $this->eventRegistry->eventNames();
+
+        $subscribers = [];
+
+        /** @var MetadataSubscriberAccessor $subscriberAccessor */
+        foreach ($this->subscriberAccessorRepository?->all() ?? [] as $subscriberAccessor) {
+            $metadata = $subscriberAccessor->metadata();
+
+            $eventsHandled = array_values(array_intersect_key($eventNames, $metadata->subscribeMethods));
```

Comment:
Here we are filtering the configured events from the subscriber by the registered events in the registry, right?

Not sure if we should do this here, as the subscriber defines that it is handling these events, even if they are not present in the registry. IMHO this would just hide information from the user. It may be a misconfiguration or intended but we should show it. WDYT @DavidBadura ?

We could think about listing these events differently, highlighting it with a warning color (yellow) if possible or in a dedicated column which indicates that these events are not registered.

Reply:
I also think that we should display what the user has specified, regardless of whether it's in the registry or not. I also think we shouldn't use highlighting here. 

If we want to include such checks, we should do it in our phpstan/psalm plugin.